packages/sui-studio/src/environment-mocha/setupEnvironment.js

```diff
@@ -1,21 +1,54 @@
-import SUIContext from '@s-ui/react-context'
+import PropTypes from 'prop-types'
 import {render} from '@testing-library/react'
 
-const setupEnvironment = (Component, context) => (props, ref) => {
+const DefaultWrapper = ({children}) => <>{children}</>
+
+DefaultWrapper.propTypes = {
+  children: PropTypes.any
+}
+
+const getWrapper = ({contexts = [], wrapper: Wrapper = DefaultWrapper}) => {
+  const ContextsWrapper = ({children}) =>
+    contexts.reduce(
+      (children, {provider: Provider, value}) => (
+        <Provider value={value}>{children}</Provider>
+      ),
+      children
+    )
+
+  const CustomWrapper = ({children}) => (
+    <ContextsWrapper>
+      <Wrapper>{children}</Wrapper>
+    </ContextsWrapper>
+  )
+
+  CustomWrapper.propTypes = {
+    children: PropTypes.any
+  }
+
+  return CustomWrapper
+}
+
+/**
+ * Render a component using React Testing Library (RTL)
+ * Keep RTL api
+ * https://testing-library.com/docs/react-testing-library/api/#render-options
+ */
+const setupEnvironment = (
+  Component,
+  {contexts = [], hydrate, queries, wrapper} = {}
```

Comment:
`{contexts = [], hydrate, queries, wrapper, ...otherSettings} = {}`

add otherSettings destructuring in render config (L46)

Reply:
RTL render method does not have more options than used on L46
Well, it has `baseElement` but if the `container` options is specified, then this defaults to that.
See: https://testing-library.com/docs/react-testing-library/api/#render-options